client/render-system.hpp

```diff
@@ -46,6 +47,8 @@ class RenderSystem :
 
 	void Update(const double delta);
 
+	bool HasExtension(const std::string& x) const { return extensions.find(x) != extensions.cend(); }
```

Comment:
Is this meant to be a file extension check? Seems out of place in this class.

Reply:
OpenGL extensions, nothing to do with files.